.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           - 11
         scala:
           - 2.11.12
-          - 2.12.12
+          - 2.12.13
```

Comment:
shouldn't we add 2.13 to this list?

Reply:
@johnynek sorry for the late reply, hard couple of days. Indeed! I guess we forgot to add it when we release 2.13.x.